core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -365,6 +365,7 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
 
     def doControlledShutdown(retries: Int): Boolean = {
       val metadataUpdater = new ManualMetadataUpdater()
+      val logContext = new LogContext("[KafkaServer brokerId=" + config.brokerId.toString + "] ")
```

Comment:
Again, we may want to do this at the same time as `logIdent` to keep things consistent.